src/components/Accordion.tsx

```diff
@@ -2,10 +2,12 @@ import { useId, useResizeObserver } from "@react-aria/utils";
 import { Dispatch, ReactNode, SetStateAction, useCallback, useEffect, useRef, useState } from "react";
 import { useFocusRing } from "react-aria";
 import { Icon } from "src/components/Icon";
-import { Css, Palette } from "src/Css";
+import { Css, Only, Padding, Palette, Xss } from "src/Css";
 import { useTestIds } from "src/utils";
 
-export interface AccordionProps {
+type AccordionXss = Xss<Padding>;
+
+export interface AccordionProps<X> {
```

Comment:
Suggestion: I think the build can be fixed by making this:

```
export interface AccordingProps<X = AccordionXss> {
```

Reply:
got it. ty!